etc/testing/travis_install.sh

```diff
@@ -1,88 +0,0 @@
-#!/bin/bash
-
-set -ex
-
-# install latest version of docker
```

Comment:
@lukemarsden how does docker get installed in the testfaster VMs, actually? Is is part of the base VM image they use? Is that based on a public docker image?

Reply:
it's baked in, the contract with testfaster is that it gives you a VM with docker installed and also the version of kubernetes specified in `kubernetes_version` in `.testfaster.yml`